internal/indexnode/task.go

```diff
@@ -346,7 +347,8 @@ func (it *IndexBuildTask) Execute(ctx context.Context) error {
 
 		getSavePathByKey := func(key string) string {
 			// TODO: fix me, use more reasonable method
-			return strconv.Itoa(int(it.req.IndexBuildID)) + "/" + strconv.Itoa(int(it.req.Version)) + "/" + strconv.Itoa(int(partitionID)) + "/" + strconv.Itoa(int(segmentID)) + "/" + key
+			return path.Join(Params.IndexRootPath, strconv.Itoa(int(it.req.IndexBuildID))+"/"+strconv.Itoa(int(it.req.Version))+"/"+
```

Comment:
you should either use path.Join()
or path + "/" + path + "/"
but not combine both~

Reply:
ok, I will fix it.